### PR TITLE
Website enhancement / altslice fix

### PR DESCRIPTION
**Pull request by echiu64**
Website - enable/disable debug from web
altslice fix 
- indent fix
- reset on LP (might not be needed with indent fix)

### CODE REVIEW

File: website.py [hunk 1]
```diff
@@ -118,6 +118,10 @@ class dynamicSite(resource.Resource):
                 except Exception,e:
                     self.bh.log_dbg('Incorrect http post reloadconfig')
                     self.bh.log_dbg(e)
+            if "enableDebug" in v:
+                self.bh.options.debug = True
+            if "disableDebug" in v:
+                self.bh.options.debug = True
```
Comment (c00w) [inline website.py:124]: Shouldn't disableDebug set debug to false?

**Comment by c00w** [comment]
Public domain?

File: website.py (hunk 1 above)
Comment (echiu64) [inline website.py:124]: Yes, sorry, pushed a earlier commit

**Comment by echiu64** [comment]
Yes public domain

**Comment by c00w** [comment]
I'm trying to merge this but I'm running into issues. Mainly which one of your branches is this from / should I merge?

**Comment by echiu64** [comment]
I've merged into a recent branch of yours, i'll send another pull

**Pull request closed without merging**